istio/istio/base/kustomization.yaml

```diff
@@ -15,4 +15,4 @@ vars:
   fieldref:
     fieldpath: data.clusterRbacConfig
 configurations:
- - params.yaml
```

Comment:
This make params.yaml not populated in the test.go. Seems this repo does have test check. Not sure how does it check in the code base.